django/core/management/commands/setup_frontend_tests.py

```diff
@@ -18,9 +18,10 @@ def handle(self, *args, **options):
         user.set_password(settings.TEST_BASIC_AUTH_PASSWORD)
         user.save()
         mp, created = MemberProfile.objects.get_or_create(user=user)
-        mp.institution = Institution.objects.get_or_create(
-            name="ASU", url="https://www.asu.edu"
-        )[0]
+        mp.affiliations = {
```

Comment:
should this be `mp.affiliations = [{ "name": "Arizona State University", ... }]`